package/setup.py

```diff
@@ -329,9 +329,24 @@ def extensions(config):
     util = MDAExtension('lib.formats.cython_util',
                         sources=['MDAnalysis/lib/formats/cython_util' + source_suffix],
                         include_dirs=include_dirs)
-
+    encore_utils = MDAExtension('analysis.encore.cutils',
```

Comment:
We adopted a new scheme that `pyx` files should be in the same folder as the resulting module. Any other c-sources that are used should be stored in subfolders called `include` and `src`. For example the affinityprop files should be stored like this.

```
analysis / 
  - encore / 
    - clustering
      - affinityprop.pyx
        caffinityprop.pxd
         include
           - ap.h
         src
           - ap.c
```

This is the same approach we use for the other cython files. It just happens to be that they currently all are in the *lib* submodule.